src/toil/test/cwl/cwlTest.py

```diff
@@ -421,6 +421,11 @@ def test_run_revsort_nochecksum(self) -> None:
             "revsort.cwl", partial(self._tester, main_args=["--no-compute-checksum"])
         )
 
+    def test_run_revsort_no_container(self) -> None:
+        self.revsort_no_checksum(
```

Comment:
We need to run the test with checksuming if we don't tell the workflow not to.
```suggestion
        self.revsort(
```